example/example.go

```diff
@@ -116,7 +117,7 @@ func main() {
 	}
 	ev.Content = strings.TrimSpace(content)
 	ev.Sign(sk)
-	for _, url := range []string{"wss://nostr.zebedee.cloud"} {
+	for _, url := range []string{"wss://nostr-relay.nokotaro.com"} {
```

Comment:
Ooooops. Sorry. This change is not need.

Reply:
It's ok.